extension/github-notifications-preview.js

```diff
@@ -145,6 +145,12 @@ async function openDropdown({currentTarget: indicator}) {
 			select.exists('.js-notifications-group', container)
 		);
 
+		let temp;
+		for (const header of select.all('.js-notifications-group h6')) {
+			temp = header.innerText.trim();
+			header.innerHTML = `<a class="text-inherit" href="${temp}">${temp}</a>`;
```

Comment:
- The `href` is missing the initial `/` from the example
- Why `innerText`? The example used `textContent`
- This is slightly unsafe, we can't concatenate user strings into `innerHTML`, Mozilla will reject it
```suggestion
		for (const header of select.all('.js-notifications-group h6')) {
			const link = doma('<a class="text-inherit"></a>');
			link.href = '/' + header.textContent.trim();
			wrap(header.firstChild, link);
```

Using `wrap`: https://github.com/sindresorhus/refined-github/blob/1fa2a55aa1d09555a10725db29ef15cd8614adda/source/helpers/dom-utils.ts#L41

Reply:
- I noticed that, but the produced links were ok.
- My mistake, sorry. After various failed attempts it missed my attention.
- Unfortunately, I'm not very experienced in JavaScript, and definitely not familiar with the various excellent utilities you have written/use in Refined GitHub, in this case `doma` and `wrap`, so I wrongly thought that a simplistic `innerHTML` was the only, unavoidable solution.

Your suggested code is perfect of course, and very educative. 
Thanks a lot for the feedback!